streams/src/main/java/org/apache/kafka/streams/state/internals/ThreadCache.java

```diff
@@ -280,6 +280,13 @@ synchronized void close(final String namespace) {
         }
     }
 
+    synchronized void clear(final String namespace) {
+        final NamedCache cleared = caches.get(namespace);
+        if (cleared != null) {
+            cleared.clear();
```

Comment:
Make sure to update `sizeInBytes` as well!

Reply:
Ack